src/ucp/proto/proto_am.inl

```diff
@@ -287,6 +300,7 @@ ucs_status_t ucp_do_am_zcopy_multi(uct_pending_req_t *self, uint8_t am_id_first,
                                          &req->send.state.uct_comp);
             } else if (state.offset == req->send.length) {
                 /* Empty IOVs on last stage */
+                ucp_am_zcopy_complete_last_stage(req, complete);
```

Comment:
why need it here? Should not we rely on completion of previous operations?

Reply:
looks like it was missed here

Reply:
the previous operation may help here, but if there is no in-progress operations we need call `complete()` here